pkg/containerruntime/docker/supported_versions.go

```diff
@@ -1,24 +1,44 @@
 package docker
 
 import (
-	"fmt"
-
 	"github.com/Masterminds/semver"
-	"github.com/kubermatic/machine-controller/pkg/containerruntime/errors"
 )
 
-// GetOfficiallySupportedVersions returns the officially supported docker version for the given kubernetes version
-func GetOfficiallySupportedVersions(kubernetesVersion string) ([]string, error) {
-	v, err := semver.NewVersion(kubernetesVersion)
-	if err != nil {
-		return nil, err
+// GetVersionsForKubelet returns the officially supported docker version for the given kubernetes version
+// The returned versions are sorted from highest to lowest
+func GetVersionsForKubelet(v *semver.Version) []string {
```

Comment:
Why not simply hardcode this in the Ubuntu userdata template as well?

Reply:
Good idea, I'll update the pr